### PR TITLE
feature: update blkio device's read/write Bps/IOps

### LINKED ISSUE #2509: [feature request] Support update container's device io limit
## Why you need it?
We can set container's block device io limit by when it is created. Such as
```
pouch run --name test3 --device-read-bps=/dev/xvda:100M registry.hub.docker.com/library/busybox:latest top
```
There are 4 device io limit options, --device-read-bps/--device-write-bps/--device-read-iops/--device-write-iops. However, we can't update them by using "pouch update" command like other container resource. I will be useful to support update them.

## How it could be?
It seems that container update API supports to update device io limit. But pouch update command doesn't.
So pouch update command could be added the four options. After that, we can update(also add and remove) container's device io limit.

### PR DESCRIPTION
**Pull request by zjumoon01**
Support update limit of block device, including read/write
bps/iops.

fix #2509 

This patch is rebased from @houstar .
related: https://github.com/alibaba/pouch/pull/2563

Signed-off-by: Wang Rui <baijia.wr@antfin.com>
Signed-off-by: Leno Hou <lenohou@gmail.com>

### CODE REVIEW

File: pkg/utils/resource.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package utils
```
Comment (HusterWan) [inline pkg/utils/resource.go]: `pkg` is a directory that putting all common packages unrelated to the core logic of pouch.

So, maybe we should not put `resource.go` to `pkg`, WDYT?

**Comment by allencloud** [comment]
CI fails with golint error:
```
pkg/utils/resource.go:11:1:warning: comment on exported function GetWeightDevice should be of the form "GetWeightDevice ..." (golint)
pkg/utils/resource.go:32:1:warning: comment on exported function GetThrottleDevice should be of the form "GetThrottleDevice ..." (golint)
```
PTAL @zjumoon01

File: pkg/utils/resource.go (hunk 1 above)
Comment (HusterWan) [reply pkg/utils/resource.go]: > just move GetThrottleDevice from mgr to ctrd.

agree with you.

File: ctrd/utils.go [hunk 2]
```diff
@@ -65,13 +66,75 @@ func resolver(authConfig *types.AuthConfig, resolverOpt docker.ResolverOptions)
 	return docker.NewResolver(options), nil
 }
 
+//GetWeightDevice Convert weight device from []*types.WeightDevice to []specs.LinuxWeightDevice
```
Comment (HusterWan) [inline ctrd/utils.go]: leave a space between `//` and the function name may be better?

Comment (zjumoon01) [reply ctrd/utils.go]: yep, you are right

**Comment by HusterWan** [comment]
LGTM, wait for the ci pass

**Pull request merged**